dev/conformance/conformance-tests/set-st-alone-mergeall.json

```diff
@@ -13,15 +13,16 @@
           "database": "projects/projectID/databases/(default)",
           "writes": [
             {
-              "transform": {
-                "document": "projects/projectID/databases/(default)/documents/C/d",
-                "fieldTransforms": [
-                  {
-                    "fieldPath": "a",
-                    "setToServerValue": "REQUEST_TIME"
-                  }
-                ]
-              }
+              "update": {
```

Comment:
I think this is not correct and needs to provide an update mask. Otherwise, it overwrites all fields on the backend.